auth-web/src/components/auth/manage-business/Certify.vue

```diff
@@ -22,13 +22,15 @@
 
 <script lang="ts">
 import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import { User } from '@/models/user'
+import { mapState } from 'vuex'
```

Comment:
please use namespace from vuex-class, we are using like this all-new codes.
can refer here 
https://github.com/bcgov/sbc-auth/blob/main/auth-web/src/views/auth/create-account/AccountSetupLanding.vue#L35

Reply:
Okay. Let me fix it.